_config/theme.yml

```diff
@@ -0,0 +1,12 @@
+
+---
+Name: accessway
+---
+SilverStripe\LoginForms\EnablerExtension:
+  login_themes:
+    - "silverstripe/login-forms:accessway"
+    - $default
+  excluded_actions: ['index', 'ping']
```

Comment:
If these are default values I think it's probably better to put them in PHP statics than YAML